src/rpc/core_rpc_server.cpp

```diff
@@ -1038,6 +1140,10 @@ namespace cryptonote
   bool core_rpc_server::on_submitblock(const COMMAND_RPC_SUBMITBLOCK::request& req, COMMAND_RPC_SUBMITBLOCK::response& res, epee::json_rpc::error& error_resp)
   {
     PERF_TIMER(on_submitblock);
+    // if (should_use_bootstrap_daemon())
```

Comment:
Same, why is this not uncommented?

Reply:
See https://github.com/monero-project/monero/pull/3165#discussion_r162804653